src/utils.ts

```diff
@@ -14,6 +14,11 @@ export function isTouchEvent(event: TouchEvent & MouseEvent) {
   );
 }
 
+export function isStepDivisible(min: number, max: number, step: number): boolean {
```

Comment:
nit: should we rename `max` to `value` ?

Reply:
nit2: you don't need to define `boolean` as the return type

Reply:
> nit: should we rename `max` to `value` ?

In one use case it's a max, in the other a value. I don't mind either way :)

> nit2: you don't need to define `boolean` as the return type

Saw one function that was implicit and one that was explicit so wasn't sure which way you preferred. Happy to remove it if you'd like.

Reply:
oh, cool